internal/config/.env

```
@@ -0,0 +1,5 @@
+TODO_PORT = 7540
```

В энве лучше не ставить пробелы
```suggestion
TODO_PORT=7540
```